src/FileFetcher.php

```diff
@@ -149,6 +165,24 @@ private function copy()
         }
     }
 
+    private function copyIncompatible()
+    {
+        $from = $this->getStateProperty('source');
+        $to = $this->getStateProperty('destination');
+
+      # 1 meg at a time, you can adjust this.
```

Comment:
Well.. you kind of can't, it's hardcoded? Also, curious why we're using this method rather than curl...

Reply:
@dafeder oops, copy and paste... removing